src/bilder/images/dagster/files/pipelines/edx-pipeline/mitxonline_edx.yaml.tmpl

```diff
@@ -10,7 +10,7 @@ resources:
   healthchecks:
     config:
       {{- if eq $env "qa" }}
-      check_id: {{ with secret "secret-data/data-qa/pipelines/edx/mitxonline/healthchecks-io-check-id" }}{{ .Data.value }}{{ end }}
+      check_id: {{ with secret "secret-data/data/pipelines/edx/mitxonline/healthchecks-io-check-id" }}{{ .Data.value }}{{ end }}
```

Comment:
So, the idea would be to get rid of the `/data/` path component entirely so that it's something like `secret-data/pipelines/edx/mitxonline/healthchecks-io-check-id` and then getting rid of the `{{ if eq $env "<env_name>" }}` blocks entirely. That also removes the need to track the env stage as a Consul key.

Reply:
Duh. 👍